### PR TITLE
[Cf-566] Ballista Remaining-tasks

### PR DESCRIPTION
**Pull request by ChandraAddala**
- Created embedded H2 database to export entries data
  - Created SCP server to test SCP'ing data
  - Wrote integration test
  - dryrun option to verify configurations.

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@shintasmith @usnavi Pls review.

File: src/test/scala/com/rackspace/feeds/ballista/CommpandProcessorITest.scala [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  override def fetch(runDate: DateTime, datacenter: String, dataSource: DataSource, maxRowLimit: String): String = {
+    val pgSQL = super.fetch(runDate, datacenter, dataSource, maxRowLimit)
+    
+    //hack: To use the same select query as the one used by the actual implementation. 
+    //Stripping out the copy part of the sql and converting some functions to h2 specific
+    val selectQuery = pgSQL.substring(pgSQL.indexOf("(") + 1, pgSQL.lastIndexOf(")"))
+    selectQuery.replace(selectQuery.substring(selectQuery.indexOf("regexp_replace"), selectQuery.indexOf(" as entrybody,")),
+                  "entrybody")
+               .replace(selectQuery.substring(selectQuery.indexOf("array_to_string"), selectQuery.indexOf(" as categories,")),
+                  "CONCAT_WS('|', ARRAY_GET(categories, 1), ARRAY_GET(categories, 2), ARRAY_GET(categories, 3), ARRAY_GET(categories, 4))")
```
Comment (ChandraAddala) [inline src/test/scala/com/rackspace/feeds/ballista/CommpandProcessorITest.scala]: crazy code here.

Comment (shintasmith) [inline src/test/scala/com/rackspace/feeds/ballista/CommpandProcessorITest.scala]: You are commenting your own code! lol!

File: src/test/scala/com/rackspace/feeds/ballista/CommpandProcessorITest.scala [hunk 2]
```diff
@@ -0,0 +1,132 @@
+package com.rackspace.feeds.ballista
```
Comment (usnavi) [inline src/test/scala/com/rackspace/feeds/ballista/CommpandProcessorITest.scala]: CommpandProcessorITest..?  Command?

**Comment by usnavi** [comment]
works for me.  Rename the file & I'm good with it.

File: src/test/resources/scripts/create-tables.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS entries;
+
+CREATE TABLE IF NOT EXISTS entries (
```
Comment (shintasmith) [inline src/test/resources/scripts/create-tables.sql]: I guess we have to be diligent in keeping this in sync with Atom Hopper

File: README.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ The app expects the configuration files to be present at /etc/cloudfeeds-ballist
          runDate is a date in the format yyyy-MM-dd. Data belonging to this date will be exported. Default value is yesterday.           
   -n <value> | --dbNames <value>
          dbNames is comma separated list of database names to be exported. Default value is all the databases configured.           
+  --dryrun
+           dryrun is a true/false flag. Set this flag to verify all the configurations. When this option is set other options are ignored.
```
Comment (shintasmith) [inline README.md]: Maybe comment to explicitly say that this will test connections to DB and remote systems?

**Comment by shintasmith** [comment]
lgtm

**Pull request merged**